ani-cli

```diff
@@ -237,13 +239,14 @@ download() {
 }
 
 play_episode() {
+    [ "$skip_intro" = 1 ] && skip_flag="$(ani-skip "$title" "$ep_no" | cut -d' ' -f1)" && skip_arg="$(ani-skip "$title" "$ep_no" | cut -d' ' -f2)"
     [ -z "$episode" ] && get_episode_url
     case "$player_function" in
         debug)
             [ -z "$ANI_CLI_NON_INTERACTIVE" ] && printf "All links:\n%s\nSelected link:\n" "$links"
             printf "%s\n" "$episode"
             ;;
-        mpv*) nohup "$player_function" --force-media-title="${allanime_title}Episode ${ep_no}" "$episode" >/dev/null 2>&1 & ;;
+        mpv*) nohup "$player_function" "$skip_flag" "$skip_arg" --force-media-title="${allanime_title}Episode ${ep_no}" "$episode" >/dev/null 2>&1 & ;;
```

Comment:
It should be:
```sh
# shellcheck disable=SC2086
mpv*) nohup "$player_function" $skip_flag --force-media-title="${allanime_title}Episode ${ep_no}" "$episode" >/dev/null 2>&1 & ;;
```

I hope this is right @CoolnsX

Reply:
@AzureOrange404 please verify it once.

Reply:
done and working